IMA-WorldHealth/bhima
# Add new cost center allocation basis items

Added new predefined cost center allocation basis items

- Num patients (cared for by a service)
- Num lab tests performed for a service

Closes https://github.com/IMA-WorldHealth/bhima/issues/5991
Closes https://github.com/IMA-WorldHealth/bhima/issues/5989

**TESTING**

- Use bhima_test
- Check Cost Centers > Allocation Keys > [Allocation Bases] button
- Check French translations
